package.json

```diff
@@ -68,7 +68,27 @@
       "website-components-playground"
     ]
   },
-  "devDependencies": {
+  "preconstruct": {
+    "packages": [
+      "packages/actions-global",
+      "packages/application-components",
+      "packages/application-config",
+      "packages/application-shell-connectors",
+      "packages/browser-history",
+      "packages/constants",
+      "packages/i18n",
+      "packages/l10n",
+      "packages/notifications",
+      "packages/permissions",
+      "packages/react-notifications",
+      "packages/sdk",
+      "packages/sentry",
+      "packages/url-utils",
+      "packages-backend/express",
+      "packages-backend/loggers"
```

Comment:
We could also specify `packages/*` and `!packages/foo` to have exclusion. At least that's what I did in the flopper.

Reply:
Yes but there are several packages to exclude. For now I think being explicit is better